node-module-system/a-wrapped-module.js

```diff
@@ -0,0 +1,5 @@
+console.log('typeof exports:', typeof exports);
+console.log('typeof require:', typeof require);
+console.log('typeof module:', typeof module);
+console.log('typeof __filename:', typeof __filename);
+console.log('typeof __dirname:', typeof __dirname);
```

Comment:
You're missing half of the `log`s:

---

![](https://user-images.githubusercontent.com/25161542/156941149-20df7ab0-fe0e-4033-bea3-e25f7ac8c6a0.png)

---

![image](https://user-images.githubusercontent.com/16143771/157150621-c3658d08-f8d1-4193-8dd3-e2d8e8ea1b3a.png)

Reply:
Oops, sorry!

Reply:
![a-wrapped-module](https://user-images.githubusercontent.com/25161542/157160757-035c0be9-3598-47a6-b4bd-0687f69c3d38.gif)